keras/layers/core.py

```diff
@@ -592,10 +592,21 @@ def antirectifier_output_shape(input_shape):
         model.add(Lambda(antirectifier,
                          output_shape=antirectifier_output_shape))
     ```
+    ```python
+        # add a layer that returns the hadamard product
+        # of the two input tensors
+
+        def hadamard_product(tensors):
+            return tensors[0]*tensors[1]
```

Comment:
It would be nice to also show multiple outputs. I suggest:

```python
out1 = tensors[0] * tensors[1]
out2 = K.mean(tensors[1], axis=-1)
return [out1, out2]
```

Reply:
okay,  i just made it into 

```python
# Function definition
def hadamard_product_sum(tensors):
    out1 = tensors[0] * tensors[1]
    out2 = K.sum(out1, axis=-1, keepdims=True)
    return [out1, out2]
```

Reply:
To show how to handle the output shape, let's not add keepdims